lib/contracts/notes/game/shuffle.masm

```diff
@@ -0,0 +1,51 @@
+proc.shuffle
+
+end
+
+begin 
```

Comment:
We are not using these procedure right ?

Reply:
Yes not yet